<!DOCTYPE html>
<html>
<head>
	<title>Capo lab</title>
	<link rel="stylesheet" type="text/css" href="css/style.css">
	<script src="js/sidebar.js"></script>
</head>
<body>
	<div id="sidebar">
		<ul>
			<li><a href="index.html">Capo lab</a></li>
			<li><a href="research.html">Research</a></li>
			<li><a href="people.html">Team</a></li>
			<li><a href="papers.html">Publications</a></li>
		</ul>
	</div>

	
<div id="content">
<h3 class="section-title underline">2023</h3>
<p align="justify">28. Huston G,  Lopez MLD, Cheng Y, King L, Duxbury LC, <b>Picard M</b>, Thomson-Laing G, Myler E, Caren C, Helbing CC, Kinnison MT, Saros JE, Gregory-Eaves I, Monchamp M-E, Wood SA, Armbrecht L, Ficetola GF, Kurte L, Von Eggers J, Brahney J, Parent G, Sakata MK, Doi H, <b>Capo E</b> (2023) Detection of fish sedimentary DNA in aquatic systems: A review of methodological challenges and future opportunities. <i>Environmental DNA</i> <a href="https://onlinelibrary.wiley.com/doi/epdf/10.1002/edn3.467" target="_blank">doi: 10.1002/edn3.467</a></p> 
<p align="justify">27. Cabrol L*, <b>Capo E</b>*, van Vliet D, von Meijenfeldt, Bertilsson S, Villanueva L, Sánchez-Andrea I, Björn E, Bravo AG, Heimbürger-Boavida L-E. Redox gradient shapes the abundance and diversity of mercury-methylating microorganisms along the water column of the Black Sea. <i>mSystems</i> (* joint first authors) <a href="https://doi.org/10.1128/msystems.00537-23" target="_blank">doi: 10.1128/msystems.00537-23</a></p>
<p align="justify">26. Williams J, Spanbauer T, Heintzman P, Blois J, <b>Capo E</b>, Goring S, Monchamp M-E, Parducci L, Von Eggers J, Alsos IG, Bowler C, Coolen M, Cullen N, Crump S, Epp L, Fernandez-Guerra A, Grimm E, Herzschuh U, Mereghetti A, Meyer M, Nota K, Pedersen MW, Perez V, Shapiro B, Stoof-Leichsenring K, Wood J. (2023) Strengthening global-change science by integrating aeDNA with paleoecoinformatics. <i>Trends in Ecology & Evolution</i> <a href="https://doi.org/10.1016/j.tree.2023.04.016" target="_blank"><u>doi: 10.1016/j.tree.2023.04.016</u></a></p></p>
<p align="justify">25. <b>Capo E</b>*, Cosio C*, Gascón Díez E, Loizeau J-L, Mendes E, Adatte T, Franzenburg S, Bravo AG. (2023) Anaerobic mercury methylators inhabit sinking particles of oxic water columns. <i>Water Research</i> (* joint first authors) <a href="https://doi.org/10.1016/j.watres.2022.119368" target="_blank"><u>doi: 10.1016/j.watres.2022.119368</u></a></p>
<p align="justify">24. Lin Q, Zhang K, McGowan S, Huang S, Xue Q, <b>Capo E</b>, Zhang C, Zhao C, Shen J. (2023) Characterization of lacustrine harmful algal blooms using multiple biomarkers: Historical processes, driving synergy and ecological shifts. <i>Water Research</i> <a href="https://doi.org/10.1016/j.watres.2023.119916" target="_blank"><u>doi: 10.1016/j.watres.2023.119916</u></a></p> 
<p align="justify">23. <b>Capo E</b>, Peterson BD, Kim M, Jones DS, Acinas SG, Amyot M, Bertilsson S, Björn E, Buck M, Cosio C, Elias D, Gilmour C, Goñi Urriza MS, Gu B, Lin H, Liu Y-R, McMahon K, Moreau JW, Pinhassi J, Podar M, Puente-Sánchez F, Sánchez P, Storck V, Tada Y, Vigneron A, Walsh D, Vandewalle-Capo M, Bravo AG, Gionfriddo C (2023). A consensus protocol for the recovery of mercury methylation genes from metagenomes. <i>Molecular Ecology Resources</i> <a href="https://doi.org/10.1111/1755-0998.13687" target="_blank"><u>doi: 10.1111/1755-0998.13687</u></a></p>
<p align="justify">22. Barouillet C, Monchamp M-E, Bertilsson S, Brasell K, Domaizon I, Epp LS, Ibrahim A, Mejbel H, Nwosu EC, Pearman JK, <b>Picard M</b>, Thomson-Laing G, Tsugeki N, Von Eggers J, Gregory-Eaves I, Pick F, Wood S, <b>Capo E</b>. (2023) Investigating the effects of anthropogenic stressors on lake biota using sedimentary DNA. <i>Freshwater Biology</i> <a href="https://doi.org/10.1111/fwb.14027" target="_blank"><u>doi: 10.1111/fwb.14027</u></a></p>
<p align="justify">21. <b>Zhong M</b>*, <b>Capo E</b>*, Zhang H, Hu H, Wang Z, Tian W, Huang T, Bertilsson S. (2023) Homogenization of water and sediment bacterial communities in a shallow lake (Lake Balihe, China). <i>Freshwater Biology</i> (* joint first authors) <a href="https://doi.org/10.1111/fwb.14016" target="_blank"><u>doi: 10.1111/fwb.14016</u></a></p>
<br>  
<h3 class="section-title underline">2022</h3>
<p align="justify">20. <b>Capo E</b>*, Feng C*, Bravo AG, Bertilsson S, Soerensen A, Pinhassi J, Buck M, Karlsson C, Hawkes J, Björn E (2022) Expression levels of hgcAB genes and mercury availability jointly explain methylmercury formation in stratified brackish waters. <i>Environmental Science & Technology</i> (* joint first authors) <a href="https://doi.org/10.1021/acs.est.2c03784" target="_blank"><u>doi: 10.1021/acs.est.2c03784</u></a></p>
<p align="justify">19. <b>Capo E</b>*, Broman E*, Bonaglia S, Bravo AG, Bertilsson S, Soerensen A, Pinhassi J, Lundin D, Buck M, Hall POJ Nascimento FJA, Björn E (2022).  Oxygen-deficient water zones in the Baltic Sea promote uncharacterized Hg methylating microbial guilds in the sediment. <i>Limnology & Oceanography</i> (* joint first authors). <a href="https://doi.org/10.1002/lno.11981" target="_blank"><u>doi: 10.1002/lno.11981</u></a></p>
<p align="justify">18. <b>Capo E</b>*, Monchamp M-E*, Coolen MJL, Domaizon I, Armbrecht L, Bertilsson S. (2022) Environmental paleomicrobiology: using DNA preserved in aquatic sediments to its full potential. <i>Environmental Microbiology</i> (* joint first authors) <a href="https://doi.org/10.1111/1462-2920.15913" target="_blank"><u>doi: 10.1111/1462-2920.15913</u></a></p><br>  
<h3 class="section-title underline">2021</h3>
<p align="justify">17. Lin Q, Zhang K, McGowan S, <b>Capo E</b>, Shen J (2021) Synergistic impacts of anthropogenic nutrient and climate change on long-term water quality and ecological dynamics in contrasting shallow-lake zones. <i>Limnology & Oceanography</i> <a href="https://doi.org/10.1002/lno.11878" target="_blank"><u>doi: 10.1002/lno.11878</u></a></p>
<p align="justify">16. Figueroa D*, <b>Capo E</b>*, Lindh MV, Rowe OF, Paczkowska J, Pinhassi J, Andersson A (2021) Terrestrial dissolved organic matter inflow drives temporal dynamics of the bacterial community of a subarctic estuary (northern Baltic Sea). <i>Environmental Microbiology</i> (* joint first authors) <a href="https://sfamjournals.onlinelibrary.wiley.com/doi/10.1111/1462-2920.15597" target="_blank"><u>doi: 10.1111/1462-2920.15597</u></a></p>
<p align="justify">15. Mattson L, Sörenson E, <b>Capo E</b>, Farnelid H, Hirwa M, Lindehoff E, Olofsson M, Svensson F, Legrand C. (2021) Functional diversity facilitates stability under environmental changes in an outdoor microalgal cultivation system. <i>Frontiers in Bioengineering and Biotechnology</i> <a href="https://doi.org/10.3389/fbioe.2021.651895" target="_blank"><u>doi: 10.3389/fbioe.2021.651895</u></a></p>
<p align="justify">14. <b>Capo E</b>, Ninnes S, Domaizon I, Bertilsson S, Bigler C, Wang X-R, Bindler R, Rydberg J. (2021) Landscape setting drives the microbial eukaryotic community structure in four Swedish mountain lakes over the Holocene. <i>Microorganisms</i> <a href="https://doi.org/10.3390/microorganisms9020355" target="_blank"><u>doi: 10.3390/microorganisms9020355</u></a></p>
<p align="justify">13. Sörenson E, <b>Capo E</b>, Farnelid H, Lindehoff E, Legrand C. (2021) Temperature stress induces shift from co-existence to competition for organic carbon in microalgae-bacterial photobioreactor community – enabling continuous production of microalgal biomass. <i>Frontiers in Microbiology</i> <a href="https://doi.org/10.3389/fmicb.2021.607601" target="_blank"><u>doi: 10.3389/fmicb.2021.607601</u></a></p>
<p align="justify">12. <b>Capo E</b>, Giguet-Covex C, Rouillard A,  Nota K, Heintzman PD, Vuillemin A, Ariztegui D, Arnaud F, Belle S, Bertilsson S, Bigler C, Bindler R, Brown AG, Clarke CL, Crump S, Debroas D, Englund G, Ficetola FG, Garner R, Gauthier J, Gregory-Eaves I, Heinecke L,  Herzschuh U, Ibrahim A, Kisand V, Kjær KH, Lammers Y, Littlefair J, Messager E, Monchamp M-E, <b>Olajos F</b>, Orsi W, Pedersen MW, Rijal D, Rydberg J, Spanbauer T, Stoof-Leichsenring K, Taberlet P, Talas L, Thomas C, Walsh D, Wang Y, Willerslev E, Woerkom A, Zimmermann H, Coolen MJL, Epp L, Domaizon I, Alsos I, Parducci L. (2021) Lake sedimentary DNA research on past terrestrial and aquatic biodiversity: Overview and recommendations. <i>Quaternary</i> <a href="https://doi.org/10.3390/quat4010006" target="_blank"><u>doi: 10.3390/quat4010006</u></a></p>
<p align="justify">11. Ibrahim A, <b>Capo E</b>, Wessels M, Martin I, Meyer A, Schleheck D, Epp L. (2021) Anthropogenic impact on the historical phytoplankton community of Lake Constance reconstructed by multimarker analysis of sediment‐core environmental DNA. <i>Molecular Ecology</i> <a href="https://doi.org/10.1111/mec.15696" target="_blank"><u>doi: 10.1111/mec.15696</u></a></p>
<p align="justify">10. <b>Capo E</b>, Spong G, Koizumi S, Puts I, <b>Olajos F</b>, Königsson H, Karlsson J, Byström P (2021). Droplet digital PCR applied to environmental DNA, a promising method to estimate fish population abundance from humic-rich aquatic ecosystems. <i>Environmental DNA</i> <a href="https://doi.org/10.1002/edn3.115" target="_blank"><u>doi: 10.1002/edn3.115</u></a></p>
<br>    
<h3 class="section-title underline">2020</h3>
<p align="justify">9. <b>Capo E</b>, Bravo AG, Soeresen A, Bertilsson S, Pinhassi J, Feng C, Andersson AF, Buck M, Björn E (2020) Deltaproteobacteria and Spirochaetes-Like Bacteria Are Abundant Putative Mercury Methylators in Oxygen-Deficient Water and Marine Particles in the Baltic Sea. <i>Frontiers in Microbiology</i> <a href="https://doi.org/10.3389/fmicb.2020.574080" target="_blank"><u>doi: 10.3389/fmicb.2020.574080</u></a></p>
<p align="justify">8. <b>Capo E</b>, Spong G, Königsson H, Byström P. (2020) Effects of filtration methods and water volume on the quantification of brown trout (Salmo trutta) and Arctic char (Salvelinus alpinus) eDNA concentrations via droplet digital PCR. <i>Environmental DNA</i> <a href="https://doi.org/10.1002/edn3.52" target="_blank"><u>doi: 10.1002/edn3.52</u></a></p>
<br>    
<h3 class="section-title underline">2019</h3>
<p align="justify">7. <b>Capo E</b>*, Rydberg J*, Tolu J*, Domaizon I, Debroas D, Bindler R, Bigler C (2019) How does environmental inter-annual variability shape aquatic microbial communities? A 40-year annual record of sedimentary DNA from a boreal lake (Nylandssjön, Sweden). <i>Frontiers in Ecology and Evolution</i> (* joint first authors) <a href="https://www.frontiersin.org/articles/10.3389/fevo.2019.00245/full" target="_blank"><u>doi: 10.3389/fevo.2019.00245</u></a></p>
<p align="justify">6. <b>Capo E</b>, Spong G, Norman S, Königsson H, Bartels P, Byström P. (2019) Droplet digital PCR assays for the quantification of brown trout (Salmo trutta) and Arctic char (Salvelinus alpinus) from environmental DNA collected in the water of mountain lakes. <i>PLOS ONE</i> <a href="https://doi.org/10.1371/journal.pone.0226638" target="_blank"><u>doi: 10.1371/journal.pone.0226638</u></a></p>
<br>
<h3 class="section-title underline">2017</h3>
<p align="justify">5. <b>Capo E</b>, Debroas D, Arnaud F, Perga ME, Chardon C, Domaizon I. (2017) Tracking a century of changes in microbial eukaryotic diversity in lakes driven by nutrient enrichment and climate warming. <i>Environmental Microbiology</i> <a href="https://doi.org/10.1111/1462-2920.13815" target="_blank"><u>doi: 10.1111/1462-2920.13815</u></a></p>
<p align="justify">4. <b>Capo E</b>*, Domaizon I*, Maier D, Debroas D, Bigler C. (2017) To what extent is the DNA of microbial eukaryotes modified during burying into lake sediments? A repeat–coring approach on annually laminated sediments. <i>Journal of Paleolimnology</i> (* joint first authors)  <a href="https://doi.org/10.1007/s10933-017-0005-9" target="_blank"><u>doi: 10.1007/s10933-017-0005-9</u></a></p>
<p align="justify">3. Domaizon I, Winegardner A, <b>Capo E</b>, Gauthier J, Gregory-Eaves I. (2017) Application of DNA-based methods in paleolimnology: new opportunities for investigating long-term dynamics of lacustrine biodiversity. <i>Journal of Paleolimnology</i> <a href="https://doi.org/10.1007/s10933-017-9958-y" target="_blank"><u>doi: 10.1007/s10933-017-9958-y</u></a></p>
<br>
<h3 class="section-title underline">2016</h3>
<p align="justify">2. <b>Capo E</b>, Debroas D, Arnaud F, Guillemot T, Bichet V, Millet L, Gauthier E, Massa C, Develle A-L, Pignol C, Lejzerowicz F, Domaizon I. (2016) Long term dynamics in microbial eukaryotes communities: a paleolimnological view based on sedimentary DNA. <i>Molecular Ecology</i> <a href="https://doi.org/10.1111/mec.13893" target="_blank"><u>doi: 10.1111/mec.13893</u></a></p>
<br>
<h3 class="section-title underline">2015</h3>
<p align="justify">1. <b>Capo E</b>, Debroas D, Arnaud F, Domaizon I. (2015). Is Planktonic diversity well recorded in sedimentary DNA? Toward the reconstruction of past protistan diversity. <i>Microbial Ecology</i> <a href="https://doi.org/10.1007/s00248-015-0627-2" target="_blank"><u>doi: 10.1007/s00248-015-0627-2</u></a></p>

</div>
</body>
</html>
