<!DOCTYPE html>
<html>
<head>
	<title>Capo lab</title>
	<link rel="stylesheet" type="text/css" href="css/style.css">
	<script src="js/sidebar.js"></script>
</head>
<body>
	<div id="sidebar">
		<ul>
			<li><a href="index.html">Capo lab</a></li>
			<li><a href="research.html">Research</a></li>
			<li><a href="people.html">Team</a></li>
			<li><a href="papers.html">Publications</a></li>
		</ul>
	</div>

	
<div id="content">
<h3 class="section-title underline">2024</h3>
<p align="justify"><b>Yan D</b>, Han Y, Liu J, Zan S, Lu Y, An Z, <b>Capo E*</b> (2024) Metagenomic analysis of sedimentary archives reveals ‘historical’ antibiotic resistance genes diversity increased over recent decades in the environment. <i>Environmental Research Letters</i> <a href="https://doi.org/" target="_blank">doi: </a></p>
<p align="justify"><b>Capo E*</b>, <b>Picard M*</b>, Nakane K, Kuwae M, Bertilsson S, Kagami M, Liu X, Sakai Y, Tsugeki N (2024) A sedimentary DNA perspective about the influence of environmental and food web changes on the microbial eukaryotic community of Lake Biwa. <i>Freshwater Biology</i> <a href="https://doi.org/10.1111/fwb.14326" target="_blank">doi: 10.1111/fwb.14326 </a></p>
<p align="justify">Sanz-Sáez I, Bravo AG, Ferri M, Carreras J-M, Sánchez O, Sebastian M, Ruiz-González C, <b>Capo E</b>, Duarte CM, Gasol JM, Sánchez P, Acinas SG (2024) Microorganisms involved in methylmercury demethylation and mercury reduction are widely distributed and active in the bathypelagic deep ocean waters. <i>Environmental Science & Technology</i> <a href="https://doi.org/10.1021%2Facs.est.4c00663" target="_blank">doi: 10.1021/acs.est.4c00663 </a></p>
<p align="justify"><b>Yan D</b>, Han Y, <b>Zhong M</b>, Wen H, An Z, <b>Capo E</b> (2024) Historical trajectories of antibiotics resistance genes assessed through sedimentary DNA analysis of a subtropical eutrophic lake. <i>Environmental International</i> <a href="https://doi.org/10.1016/j.envint.2024.108654" target="_blank">doi: 10.1016/j.envint.2024.108654</a></p>
<p align="justify">Lin Q, Zhang K, Giguet-Covex C, Arnaud F, McGowan S, Gielly L, <b>Capo E</b>, Huang S, Ficetola GF, Shen J, Dearing JA, Meadows ME (2024) Transient social-ecological dynamics reveal signals of decoupling in a highly disturbed Anthropocene landscape. <i>Proceedings of the National Academy of Sciences</i> <a href="https://doi.org/10.1073/pnas.2321303121" target="_blank">doi: 10.1073/pnas.2321303121</a></p>
<p align="justify"><b>Yan D</b>, <b>Picard M</b>, Han Y, An Z, Lei D, Zhao X, Zhang L, <b>Capo E</b> (2024) Sedimentary DNA reveals phytoplankton diversity loss in a deep maar lake during the Anthropocene. <i>Limnology & Oceanography</i> <a href="http://doi.org/10.1002/lno.12562" target="_blank">doi: 10.1002/lno.12562</a></p>
<p align="justify">Von Eggers J, Wisnoki NI, Calder JW, <b>Capo E</b>, Groff DV, Krist A, Shuman B (2024) Environmental filtering governs consistent vertical zonation in sedimentary microbial communities across disconnected mountain lakes. <i>Environmental Microbiology</i>. <a href="https://doi.org/10.1111/1462-2920.16607" target="_blank">doi: 10.1111/1462-2920.16607</a></p>
<p align="justify"><b>Yan D</b>, Han Y, An Z, Lei D, Zhao X, Zhao H, Liu J, <b>Capo E</b> (2024) Anthropogenic drivers accelerate the changes of lake microbial eukaryotic communities over the past 160 years. <i>Quaternary Science Reviews</i>. <a href=" https://doi.org/10.1016/j.quascirev.2024.108535" target="_blank">doi: 10.1016/j.quascirev.2024.108535</a></p>
<p align="justify"><b>Picard M</b>, Von Eggers J, Brasell KA, <b>Yan D</b>, Klaminder J, Alsos IG, Barouillet C, Cheng Y, Dommain R, Dulias K, Duxbury L, Edwards ME, Garcés Pastor S, Harning D, Hudson SM, Hutson G, Kaynar S, Kurte L, Leunda M, Lopez MLD, Moguel B, <b>Olajos F</b>, Pérez Godoy V, Revéret A, Rijal DP, Rydberg J, Schwörer C, Stoof-Leichsenring KR, Taranu ZE, Thomson-Laing G, Thorpe A, Tiedemann R, Nogales LV, Wang Y, Wood SA, <b>Capo E</b> (2024) Using DNA archived in lake sediments to reconstruct past ecosystems. Chapter in the Encyclopedia of Quaternary Science, 3rd edition <a href="https://doi.org/10.1016/B978-0-323-99931-1.00171-9" target="_blank">doi: 10.1016/B978-0-323-99931-1.00171-9 </a></p> 
<p align="justify"><b>Yan D</b>, An Z, <b>Capo E</b> (2024) Organic matter content and source is associated with the depth-dependent distribution of prokaryotes in lake sediments. <i>Freshwater Biology</i> <a href="https://doi.org/10.1111/fwb.14223" target="_blank">doi: 10.1111/fwb.14223</a></p> 
<p align="justify">Rincón-Tomás B, Lanzén A, Sánchez P, Estupiñán M, Sanz-Saéz I, Elisabete Bilbao M, Rojo D, Mendibil I, Pérez-Cruz C, Ferri M, <b>Capo E</b>, Abad-Recio IL, Amouroux D, Bertilsson S, Sánchez O, Acinas SG, Alonso-Sáez L (2024) Revisiting the mercury cycle in marine sediments: a potential multifaceted role for Desulfobacterota. <i>Journal of Hazardous Materials</i> <a href="https://www.sciencedirect.com/science/article/pii/S0304389423024044" target="_blank">doi: 10.1016/j.jhazmat.2023.133120</a></p> 	
<br>  
<h3 class="section-title underline">2023</h3>
<p align="justify"><b>Capo E</b>, Barouillet C, Smol JP (2023). Tracking environmental change using lake sediments: Sedimentary DNA (Vol. 6). Springer Cham <a href="https://doi.org/10.1007/978-3-031-43799-1" target="_blank"><u>doi: 10.1007/978-3-031-43799-1</u></a></p>
<p align="justify"><b>Capo E</b>, Barouillet C, Smol JP (2023). Using lake sedimentary DNA to reconstruct biodiversity changes. Chapter in the book “Tracking Environmental change using lake sediments: Sedimentary DNA (Vol. 6). Springer Cham (eds Capo E, Barouillet C, Smol JP). <a href="https://doi.org/10.1007/978-3-031-43799-1_1" target="_blank"><u>doi: 10.1007/978-3-031-43799-1_1</u></a></p>
<p align="justify">Giguet-Covex C, Jelavić S, Foucher A, Morlock MA, Wood SA, Augustijns F, Domaizon I, Gielly L, <b>Capo E</b> (2023). The sources and fates of lake sedimentary DNA. Chapter in the book “Tracking Environmental change using lake sediments: Sedimentary DNA (Vol. 6). Springer Cham (eds Capo E, Barouillet C, Smol JP). <a href="https://doi.org/10.1007/978-3-031-43799-1_2" target="_blank"><u>doi: 10.1007/978-3-031-43799-1_2</u></a></p>
<p align="justify">Barouillet C, Domaizon I, <b>Capo E</b> (2023). Protist DNA from lake sediments. Chapter in the book “Tracking Environmental change using lake sediments: Sedimentary DNA (Vol. 6). Springer Cham (eds Capo E, Barouillet C, Smol JP). <a href="https://doi.org/10.1007/978-3-031-43799-1_6" target="_blank"><u>doi: 10.1007/978-3-031-43799-1_6</u></a></p>
<p align="justify">Huston G,  Lopez MLD, Cheng Y, King L, Duxbury LC, <b>Picard M</b>, Thomson-Laing G, Myler E, Caren C, Helbing CC, Kinnison MT, Saros JE, Gregory-Eaves I, Monchamp M-E, Wood SA, Armbrecht L, Ficetola GF, Kurte L, Von Eggers J, Brahney J, Parent G, Sakata MK, Doi H, <b>Capo E</b> (2023) Detection of fish sedimentary DNA in aquatic systems: A review of methodological challenges and future opportunities. <i>Environmental DNA</i> <a href="https://onlinelibrary.wiley.com/doi/epdf/10.1002/edn3.467" target="_blank">doi: 10.1002/edn3.467</a></p> 
<p align="justify">Cabrol L*, <b>Capo E</b>*, van Vliet D, von Meijenfeldt, Bertilsson S, Villanueva L, Sánchez-Andrea I, Björn E, Bravo AG, Heimbürger-Boavida L-E. Redox gradient shapes the abundance and diversity of mercury-methylating microorganisms along the water column of the Black Sea. <i>mSystems</i> (* joint first authors) <a href="https://doi.org/10.1128/msystems.00537-23" target="_blank">doi: 10.1128/msystems.00537-23</a></p>
<p align="justify">Williams J, Spanbauer T, Heintzman P, Blois J, <b>Capo E</b>, Goring S, Monchamp M-E, Parducci L, Von Eggers J, Alsos IG, Bowler C, Coolen M, Cullen N, Crump S, Epp L, Fernandez-Guerra A, Grimm E, Herzschuh U, Mereghetti A, Meyer M, Nota K, Pedersen MW, Perez V, Shapiro B, Stoof-Leichsenring K, Wood J. (2023) Strengthening global-change science by integrating aeDNA with paleoecoinformatics. <i>Trends in Ecology & Evolution</i> <a href="https://doi.org/10.1016/j.tree.2023.04.016" target="_blank"><u>doi: 10.1016/j.tree.2023.04.016</u></a></p></p>
<p align="justify"><b>Capo E</b>*, Cosio C*, Gascón Díez E, Loizeau J-L, Mendes E, Adatte T, Franzenburg S, Bravo AG. (2023) Anaerobic mercury methylators inhabit sinking particles of oxic water columns. <i>Water Research</i> (* joint first authors) <a href="https://doi.org/10.1016/j.watres.2022.119368" target="_blank"><u>doi: 10.1016/j.watres.2022.119368</u></a></p>
<p align="justify">Lin Q, Zhang K, McGowan S, Huang S, Xue Q, <b>Capo E</b>, Zhang C, Zhao C, Shen J. (2023) Characterization of lacustrine harmful algal blooms using multiple biomarkers: Historical processes, driving synergy and ecological shifts. <i>Water Research</i> <a href="https://doi.org/10.1016/j.watres.2023.119916" target="_blank"><u>doi: 10.1016/j.watres.2023.119916</u></a></p> 
<p align="justify"><b>Capo E</b>, Peterson BD, Kim M, Jones DS, Acinas SG, Amyot M, Bertilsson S, Björn E, Buck M, Cosio C, Elias D, Gilmour C, Goñi Urriza MS, Gu B, Lin H, Liu Y-R, McMahon K, Moreau JW, Pinhassi J, Podar M, Puente-Sánchez F, Sánchez P, Storck V, Tada Y, Vigneron A, Walsh D, Vandewalle-Capo M, Bravo AG, Gionfriddo C (2023). A consensus protocol for the recovery of mercury methylation genes from metagenomes. <i>Molecular Ecology Resources</i> <a href="https://doi.org/10.1111/1755-0998.13687" target="_blank"><u>doi: 10.1111/1755-0998.13687</u></a></p>
<p align="justify">Barouillet C, Monchamp M-E, Bertilsson S, Brasell K, Domaizon I, Epp LS, Ibrahim A, Mejbel H, Nwosu EC, Pearman JK, <b>Picard M</b>, Thomson-Laing G, Tsugeki N, Von Eggers J, Gregory-Eaves I, Pick F, Wood S, <b>Capo E</b>. (2023) Investigating the effects of anthropogenic stressors on lake biota using sedimentary DNA. <i>Freshwater Biology</i> <a href="https://doi.org/10.1111/fwb.14027" target="_blank"><u>doi: 10.1111/fwb.14027</u></a></p>
<p align="justify"><b>Zhong M</b>*, <b>Capo E</b>*, Zhang H, Hu H, Wang Z, Tian W, Huang T, Bertilsson S. (2023) Homogenization of water and sediment bacterial communities in a shallow lake (Lake Balihe, China). <i>Freshwater Biology</i> (* joint first authors) <a href="https://doi.org/10.1111/fwb.14016" target="_blank"><u>doi: 10.1111/fwb.14016</u></a></p>
<br>  
<h3 class="section-title underline">2022</h3>
<p align="justify"><b>Capo E</b>*, Feng C*, Bravo AG, Bertilsson S, Soerensen A, Pinhassi J, Buck M, Karlsson C, Hawkes J, Björn E (2022) Expression levels of hgcAB genes and mercury availability jointly explain methylmercury formation in stratified brackish waters. <i>Environmental Science & Technology</i> (* joint first authors) <a href="https://doi.org/10.1021/acs.est.2c03784" target="_blank"><u>doi: 10.1021/acs.est.2c03784</u></a></p>
<p align="justify"><b>Capo E</b>*, Broman E*, Bonaglia S, Bravo AG, Bertilsson S, Soerensen A, Pinhassi J, Lundin D, Buck M, Hall POJ Nascimento FJA, Björn E (2022).  Oxygen-deficient water zones in the Baltic Sea promote uncharacterized Hg methylating microbial guilds in the sediment. <i>Limnology & Oceanography</i> (* joint first authors). <a href="https://doi.org/10.1002/lno.11981" target="_blank"><u>doi: 10.1002/lno.11981</u></a></p>
<p align="justify"><b>Capo E</b>*, Monchamp M-E*, Coolen MJL, Domaizon I, Armbrecht L, Bertilsson S. (2022) Environmental paleomicrobiology: using DNA preserved in aquatic sediments to its full potential. <i>Environmental Microbiology</i> (* joint first authors) <a href="https://doi.org/10.1111/1462-2920.15913" target="_blank"><u>doi: 10.1111/1462-2920.15913</u></a></p><br>  
<br> 
<h3 class="section-title underline">2021</h3>
<p align="justify">Lin Q, Zhang K, McGowan S, <b>Capo E</b>, Shen J (2021) Synergistic impacts of anthropogenic nutrient and climate change on long-term water quality and ecological dynamics in contrasting shallow-lake zones. <i>Limnology & Oceanography</i> <a href="https://doi.org/10.1002/lno.11878" target="_blank"><u>doi: 10.1002/lno.11878</u></a></p>
<p align="justify">Figueroa D*, <b>Capo E</b>*, Lindh MV, Rowe OF, Paczkowska J, Pinhassi J, Andersson A (2021) Terrestrial dissolved organic matter inflow drives temporal dynamics of the bacterial community of a subarctic estuary (northern Baltic Sea). <i>Environmental Microbiology</i> (* joint first authors) <a href="https://sfamjournals.onlinelibrary.wiley.com/doi/10.1111/1462-2920.15597" target="_blank"><u>doi: 10.1111/1462-2920.15597</u></a></p>
<p align="justify">Mattson L, Sörenson E, <b>Capo E</b>, Farnelid H, Hirwa M, Lindehoff E, Olofsson M, Svensson F, Legrand C. (2021) Functional diversity facilitates stability under environmental changes in an outdoor microalgal cultivation system. <i>Frontiers in Bioengineering and Biotechnology</i> <a href="https://doi.org/10.3389/fbioe.2021.651895" target="_blank"><u>doi: 10.3389/fbioe.2021.651895</u></a></p>
<p align="justify"><b>Capo E</b>, Ninnes S, Domaizon I, Bertilsson S, Bigler C, Wang X-R, Bindler R, Rydberg J. (2021) Landscape setting drives the microbial eukaryotic community structure in four Swedish mountain lakes over the Holocene. <i>Microorganisms</i> <a href="https://doi.org/10.3390/microorganisms9020355" target="_blank"><u>doi: 10.3390/microorganisms9020355</u></a></p>
<p align="justify">Sörenson E, <b>Capo E</b>, Farnelid H, Lindehoff E, Legrand C. (2021) Temperature stress induces shift from co-existence to competition for organic carbon in microalgae-bacterial photobioreactor community – enabling continuous production of microalgal biomass. <i>Frontiers in Microbiology</i> <a href="https://doi.org/10.3389/fmicb.2021.607601" target="_blank"><u>doi: 10.3389/fmicb.2021.607601</u></a></p>
<p align="justify"><b>Capo E</b>, Giguet-Covex C, Rouillard A,  Nota K, Heintzman PD, Vuillemin A, Ariztegui D, Arnaud F, Belle S, Bertilsson S, Bigler C, Bindler R, Brown AG, Clarke CL, Crump S, Debroas D, Englund G, Ficetola FG, Garner R, Gauthier J, Gregory-Eaves I, Heinecke L,  Herzschuh U, Ibrahim A, Kisand V, Kjær KH, Lammers Y, Littlefair J, Messager E, Monchamp M-E, <b>Olajos F</b>, Orsi W, Pedersen MW, Rijal D, Rydberg J, Spanbauer T, Stoof-Leichsenring K, Taberlet P, Talas L, Thomas C, Walsh D, Wang Y, Willerslev E, Woerkom A, Zimmermann H, Coolen MJL, Epp L, Domaizon I, Alsos I, Parducci L. (2021) Lake sedimentary DNA research on past terrestrial and aquatic biodiversity: Overview and recommendations. <i>Quaternary</i> <a href="https://doi.org/10.3390/quat4010006" target="_blank"><u>doi: 10.3390/quat4010006</u></a></p>
<p align="justify">Ibrahim A, <b>Capo E</b>, Wessels M, Martin I, Meyer A, Schleheck D, Epp L. (2021) Anthropogenic impact on the historical phytoplankton community of Lake Constance reconstructed by multimarker analysis of sediment‐core environmental DNA. <i>Molecular Ecology</i> <a href="https://doi.org/10.1111/mec.15696" target="_blank"><u>doi: 10.1111/mec.15696</u></a></p>
<p align="justify"><b>Capo E</b>, Spong G, Koizumi S, Puts I, <b>Olajos F</b>, Königsson H, Karlsson J, Byström P (2021). Droplet digital PCR applied to environmental DNA, a promising method to estimate fish population abundance from humic-rich aquatic ecosystems. <i>Environmental DNA</i> <a href="https://doi.org/10.1002/edn3.115" target="_blank"><u>doi: 10.1002/edn3.115</u></a></p>
<br>    
<h3 class="section-title underline">2020</h3>
<p align="justify"><b>Capo E</b>, Bravo AG, Soeresen A, Bertilsson S, Pinhassi J, Feng C, Andersson AF, Buck M, Björn E (2020) Deltaproteobacteria and Spirochaetes-Like Bacteria Are Abundant Putative Mercury Methylators in Oxygen-Deficient Water and Marine Particles in the Baltic Sea. <i>Frontiers in Microbiology</i> <a href="https://doi.org/10.3389/fmicb.2020.574080" target="_blank"><u>doi: 10.3389/fmicb.2020.574080</u></a></p>
<p align="justify"><b>Capo E</b>, Spong G, Königsson H, Byström P. (2020) Effects of filtration methods and water volume on the quantification of brown trout (Salmo trutta) and Arctic char (Salvelinus alpinus) eDNA concentrations via droplet digital PCR. <i>Environmental DNA</i> <a href="https://doi.org/10.1002/edn3.52" target="_blank"><u>doi: 10.1002/edn3.52</u></a></p>
<p align="justify">Vandewalle-Capo M, <b>Capo E</b>, Rehamnia B, Sheldrake M, Lee N (2020). The biotechnological potential of yeast from extreme habitats. Chapter in the book “Biotechnological Applications of Extremophilic Microorganisms (Life in Extreme Environments), Editor: N Lee. <a href="https://doi.org/10.1515/9783110424331-013" target="_blank"><u>doi: 10.1515/9783110424331-013</u></a></p>
<br>    
<h3 class="section-title underline">2019</h3>
<p align="justify"><b>Capo E</b>*, Rydberg J*, Tolu J*, Domaizon I, Debroas D, Bindler R, Bigler C (2019) How does environmental inter-annual variability shape aquatic microbial communities? A 40-year annual record of sedimentary DNA from a boreal lake (Nylandssjön, Sweden). <i>Frontiers in Ecology and Evolution</i> (* joint first authors) <a href="https://www.frontiersin.org/articles/10.3389/fevo.2019.00245/full" target="_blank"><u>doi: 10.3389/fevo.2019.00245</u></a></p>
<p align="justify"><b>Capo E</b>, Spong G, Norman S, Königsson H, Bartels P, Byström P. (2019) Droplet digital PCR assays for the quantification of brown trout (Salmo trutta) and Arctic char (Salvelinus alpinus) from environmental DNA collected in the water of mountain lakes. <i>PLOS ONE</i> <a href="https://doi.org/10.1371/journal.pone.0226638" target="_blank"><u>doi: 10.1371/journal.pone.0226638</u></a></p>
<br>
<h3 class="section-title underline">2017</h3>
<p align="justify"><b>Capo E</b>, Debroas D, Arnaud F, Perga ME, Chardon C, Domaizon I. (2017) Tracking a century of changes in microbial eukaryotic diversity in lakes driven by nutrient enrichment and climate warming. <i>Environmental Microbiology</i> <a href="https://doi.org/10.1111/1462-2920.13815" target="_blank"><u>doi: 10.1111/1462-2920.13815</u></a></p>
<p align="justify"><b>Capo E</b>*, Domaizon I*, Maier D, Debroas D, Bigler C. (2017) To what extent is the DNA of microbial eukaryotes modified during burying into lake sediments? A repeat–coring approach on annually laminated sediments. <i>Journal of Paleolimnology</i> (* joint first authors)  <a href="https://doi.org/10.1007/s10933-017-0005-9" target="_blank"><u>doi: 10.1007/s10933-017-0005-9</u></a></p>
<p align="justify">Domaizon I, Winegardner A, <b>Capo E</b>, Gauthier J, Gregory-Eaves I. (2017) Application of DNA-based methods in paleolimnology: new opportunities for investigating long-term dynamics of lacustrine biodiversity. <i>Journal of Paleolimnology</i> <a href="https://doi.org/10.1007/s10933-017-9958-y" target="_blank"><u>doi: 10.1007/s10933-017-9958-y</u></a></p>
<br>
<h3 class="section-title underline">2016</h3>
<p align="justify"><b>Capo E</b>, Debroas D, Arnaud F, Guillemot T, Bichet V, Millet L, Gauthier E, Massa C, Develle A-L, Pignol C, Lejzerowicz F, Domaizon I. (2016) Long term dynamics in microbial eukaryotes communities: a paleolimnological view based on sedimentary DNA. <i>Molecular Ecology</i> <a href="https://doi.org/10.1111/mec.13893" target="_blank"><u>doi: 10.1111/mec.13893</u></a></p>
<br>
<h3 class="section-title underline">2015</h3>
<p align="justify"><b>Capo E</b>, Debroas D, Arnaud F, Domaizon I. (2015). Is Planktonic diversity well recorded in sedimentary DNA? Toward the reconstruction of past protistan diversity. <i>Microbial Ecology</i> <a href="https://doi.org/10.1007/s00248-015-0627-2" target="_blank"><u>doi: 10.1007/s00248-015-0627-2</u></a></p>

</div>
</body>
</html>
